.github/workflows/build-deploy.yaml

```diff
@@ -71,6 +71,27 @@ jobs:
                   echo ${GITHUB_TOKEN} | docker login docker.pkg.github.com -u ${GITHUB_REPOSITORY} --password-stdin
                   docker build -t ${IMAGE}:${IMAGE_TAG} .
                   docker push ${IMAGE}:${IMAGE_TAG}
+    update-gh-pages:
+        name: Update GitHub pages
+        runs-on: ubuntu-latest
+        #if: github.ref == 'refs/heads/dev'
```

Comment:
Skal denne med?

Reply:
Den skal vel speile master? Eventuelt også ha en dispatch så vi kan deploye direkte til demo-appen

Reply:
Its fixed, now github pages are only generated for master